app/models/metric/chargeback_helper.rb

```diff
@@ -19,14 +19,12 @@ def tag_prefix
   end
 
   def tag_list_reconstruct
-    tag_list = tag_names
-
     if resource.instance_of?(Container)
       state = resource.vim_performance_state_for_ts(timestamp.to_s)
-      tag_list += state.image_tag_names if state.present?
+      image_tag_name = "#{state.try(:image_tag_name)}|" if state
```

Comment:
yes, I think we also avoid to use `try` because then you can easily overlook that there is typo